src/Common/src/Interop/Windows/SChannel/Interop.Sec_Application_Protocols.cs

```diff
@@ -38,27 +36,23 @@ public static unsafe byte[] ToByteArray(List<SslApplicationProtocol> application
             }
 
             Sec_Application_Protocols protocols = new Sec_Application_Protocols();
-            protocols.ProtocolListsSize = (uint)(ProtocolListConstSize + protocolListSize);
-            protocols.ProtocolExtenstionType = ApplicationProtocolNegotiationExt.ALPN;
+            protocols.ProtocolListsSize = (uint)((sizeof(Sec_Application_Protocols) - sizeof(uint)) + protocolListSize);
```

Comment:
The previous code was semi-self-documenting as to what was being calculated.  That's lost a bit here.  Can you add a comment?